adbc.h

```diff
@@ -438,45 +425,7 @@ struct AdbcErrorDetail AdbcErrorGetDetail(struct AdbcError* error, int index);
 /// The type is char*.
 ///
 /// \since ADBC API revision 1.1.0
-/// \addtogroup adbc-1.1.0
 #define ADBC_OPTION_PASSWORD "password"
-/// \brief Canonical option name for error details.
-///
-/// Should be used as the expected option name to retrieve error
-/// details from the driver.  This allows drivers to return custom,
-/// structured error information (for example, JSON or Protocol
-/// Buffers) that can be optionally parsed by clients, beyond the
-/// standard AdbcError fields, without having to encode it in the
-/// error message.  The encoding of the data is driver-defined.
-/// Drivers may provide multiple error details.
-///
```

Comment:
was this intended to be removed?

Reply:
yes, that was the whole point of the past few days of discussion...

Reply:
Ah sorry. A bit discombobulated, this all makes sense now haha